taskcluster/ci/ui-test/kind.yml

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - toolchain
+    - build
+
+job-defaults:
+    attributes:
+        build-type: debug
+        code-review: true
+        retrigger: true
+    # dependencies:
+    #     build-debug: build-focus-debug
+    #     build-android-test: build-android-test-debug
+    fetches:
+        toolchain:
+            - android-sdk-linux
+            - android-gradle-dependencies
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: ui-tests}
+        max-run-time: 7200
+        env:
+            GOOGLE_APPLICATION_CREDENTIALS: '.firebase_token.json'
+            GOOGLE_PROJECT: 'moz-fx-mobile-firebase-testlab'
```

Comment:
this is the old GCP project and won't work.  Should be:
GOOGLE_PROJECT: 'moz-focus-android'